public/catalog/v0.1.0/openapi.json

```diff
@@ -297,6 +297,124 @@
         }
       }
     },
+    "/portfolios/{portfolio_id}/share": {
+      "post": {
+        "tags": [
+          "admins"
+        ],
+        "summary": "Share a portfolio with one or more groups with specific permission",
+        "operationId": "sharePortfolio",
+        "description": "Share a Portfolio with one or more groups with specific permissions\n",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/PortfolioID"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully shared the portfolio"
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "The Portfolio Object or or one or more of the groups was not found."
+          }
+        },
+        "requestBody": {
```

Comment:
Very small nit, but our current procedure has been to include the request body directly underneath the parameters key, and close out the entry with responses.